providers/infobip/src/lib/infobip.provider.spec.ts

```diff
@@ -0,0 +1,3 @@
+import { InfobipSmsProvider } from './infobip.provider';
+
+test('should trigger infobip library correctly', async () => {});
```

Comment:
Do you think it will be possible to add a simple assertion test here? Nothing fancy, but just so we can make sure that we are passing the variables needed.

Reply:
Added, is it worth me raising an issue/PR against the docs